### PR TITLE
feat: latest interop fixes

### LINKED ISSUE #144: Interop issues to be addressed
- wrong parsing of issuance Date & expiration Date
- issuer with images are incorrectly parsed
- holder based VC list filtering

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- Holder based dynamic VC table filtering
 - Holder info will be available in generated presentations
 - recent aries upgrade containing date parsing & issuer image fixes
 - Closes #144

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

File: cmd/user-agent/src/pages/Interop.vue [hunk 1]
```diff
@@ -119,7 +119,24 @@ SPDX-License-Identifier: Apache-2.0
...
                 this.clearResults()
-                const credentialQuery = JSON.parse('{"web": {"VerifiableCredential": {}}}');
+                const credentialQuery = {
+                    "web": {
+                        "VerifiableCredential": {
+                            query: [
+                                {
+                                    type: "QueryByExample",
+                                    credentialQuery: {
+                                        reason: "Please present a credential for JaneDoe.",
```
Comment (troyronda) [inline cmd/user-agent/src/pages/Interop.vue:129]: JaneDoe?

File: cmd/user-agent/src/pages/Interop.vue [hunk 2]
```diff
@@ -130,7 +147,26 @@ SPDX-License-Identifier: Apache-2.0
...
                 this.clearResults()
-                const credentialQuery = JSON.parse('{"web": {"VerifiablePresentation": {}}}');
+                const credentialQuery = {
+                    web: {
+                        VerifiablePresentation: {
+                            query: [
+                                {
+                                    type: "QueryByExample",
+                                    credentialQuery: {
+                                        reason: "Please present a credential for JaneDoe.",
```
Comment (troyronda) [inline cmd/user-agent/src/pages/Interop.vue:157]: JaneDoe?

File: cmd/user-agent/src/pages/Interop.vue (hunk 1 above)
Comment (sudeshrshetty) [reply cmd/user-agent/src/pages/Interop.vue:129]: this is test page, now exposed in dashboard

File: cmd/user-agent/src/pages/Interop.vue (hunk 2 above)
Comment (sudeshrshetty) [reply cmd/user-agent/src/pages/Interop.vue:157]: this is test page, not exposed in dashboard

File: cmd/user-agent/src/pages/Interop.vue (hunk 1 above)
Comment (sudeshrshetty) [reply cmd/user-agent/src/pages/Interop.vue:129]: used it from sample docs

**Pull request merged**